### DISCUSSION TITLE
License Query: Offering Hosted and Customized Budibase Services

### DISCUSSION THREAD

**Discussion by Yakito** [Budibase/budibase#10938] [General]
Hello everyone,

I hope this message finds you well. I am currently seeking advice regarding the type of license required for a my use-case involving Budibase.

I intend to offer a service wherein I would set up a hosted version of Budibase for my clients and customize it according to their specific needs. The customization process would primarily include tasks such as loading their data and setting up the screens.
I'm unsure about what type of license this use-case falls under. I'd like to ensure that I am in complete compliance with Budibase's licensing terms.

Would it fall under an open-source license or do I need to apply for a commercial license? Or perhaps is there another specific license that is more suited to this type of service?

I appreciate any guidance or pointers you can provide on this matter. Thanks in advance for your help.

**Comment by ConorWebb96** [comment]
Hey @Yakito,

I'd recommend clicking the contact sales button on the Budibase website https://budibase.com/. You can find this in the top right hand side of the website.
<img width="1512" alt="Screenshot 2023-06-19 at 08 32 13" src="https://github.com/Budibase/budibase/assets/126772285/6ef807de-cb41-4c09-bd6f-4a361d9881d4">


A member of the sales team will then reach out, setup a call to discuss your use case and advise you.
